packages/ethereum/contracts/HoprNetworkRegistry.sol

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/access/Ownable.sol';
+import './IHoprNetworkRegistryRequirement.sol';
+
+/**
+ * @dev Allowlist addresses and hopr nodes
+ */
+contract HoprNetworkRegistry is Ownable {
```

Comment:
Please document all attributes and events.

Reply:
Done